swagger.yml

```diff
@@ -1022,35 +1043,48 @@ components:
         Microsoft/SQLServer:
           type: boolean
           nullable: true
+        example-1: {}
     TechnologyStatus:
       type: object
-      required:
-        - Product
-        - HostsCount
-        - Compliance
-        - UnpaidDues
       properties:
         Product:
           type: string
+          description: technologial product code
         HostsCount:
           type: integer
+          description: number of hosts found supporting the current product / total hosts found
```

Comment:
What should the affermation "/ total hosts found" mean?

Reply:
seems to me it means OR but it is better to remove. done.